### PR TITLE
Drug discovery and chemical representation

### PR DESCRIPTION
**Pull request by agitter**
Work in progress.  Ref #689.

I plan to add a few new references on graph convolutions (without much discussion) to complement our new section on PPI graphs in #638.  This also adds a reference to partially address the [bioRxiv comment](http://www.biorxiv.org/content/early/2017/05/28/142760#comment-3444535593) per #654.

### CODE REVIEW

**Comment by agitter** [comment]
@dhimmel ef2b76c6612aa68c741783c5388e420bef21acad passed the build yesterday but failed today when I reran it to try to diagnose my more recent build failures.

Have you seen this error before:
```
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/manubot/bin/pandoc-fignos", line 11, in <module>
    sys.exit(main())
  File "/home/travis/miniconda/envs/manubot/lib/python3.6/site-packages/pandoc_fignos.py", line 415, in main
    plusname, starname, 'figure')
TypeError: replace_refs_factory() missing 1 required positional argument: 'target'
Error running filter pandoc-fignos:
Filter returned error status 1
```

**Comment by agitter** [comment]
This may need some additional work but is ready for feedback and review.  I didn't go into detail about the new graph methods because the reviewer already thinks this section is too technical.  I primarily sought to separate the representation learning section to break up the long ligand-based subsection and emphasize the graph-based methods.  These were requested by the same reviewer and are interesting algorithms beyond the virtual screening domain.

I could cut some text if we'd like to shorten this.

This PR adds #555, #641, #741, #762, #773, and #776.

**Comment by dhimmel** [comment]
Nothing stands out to me as problematic in this PR's diff.

@tomduck (creator of `pandoc-fignos`), any idea what [this error](https://travis-ci.org/greenelab/deep-review/builds/328724475#L775) could be caused by:

```
TypeError: replace_refs_factory() missing 1 required positional argument: 'target'
Error running filter pandoc-fignos:
```

We recently upgraded pandoc-fignos, pandoc-tablenos, and pandoc-egnos to v1.0 in https://github.com/greenelab/deep-review/commit/6c2020f8b8fef9c72dba122ecf378499e3db3c53. @agitter do you know whether the successful builds used the older version of `pandoc-fignos`?

**Comment by tomduck** [comment]
The API for pandoc-xnos was slightly changed.  You need to update it as well.

**Comment by tomduck** [comment]
Actually, it looks like you may be using the new pandoc-xnos but an old pandoc-fignos.  Can you upgrade both to the most recent version?

Note that I am in the process of preparing another release, which will go out within the next hour.

**Comment by dhimmel** [comment]
Looks like our [build](https://travis-ci.org/greenelab/deep-review/builds/328724475#L615) used `pandoc-eqnos==1.0.0`, `pandoc-fignos==1.0.0`, `pandoc-tablenos==1.0.0`, and `pandoc-xnos==0.15`.

> Actually, it looks like you may be using the new pandoc-xnos but an old pandoc-fignos. Can you upgrade both to the most recent version?

Will do soon, after the next release is out. I think the issue may be that we pinned fignos, egnos, and tablenos, but not xnos. Therefore, the error would have first occurred when a higher version pandoc-xnos started getting installed.

**Comment by tomduck** [comment]
Yes, that makes sense @dhimmel.  Is there something I should be doing on my end to prevent such a problem in the future?

**Comment by dhimmel** [comment]
As per [here](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies), I think you could specify something like `pandoc-xnos>=0.13,<0.15`, in [`setup.py`](https://github.com/tomduck/pandoc-fignos/blob/847125228bd2ec8dc0c2ad08804415da81a64b1f/setup.py#L111) of pandoc-fignos. However, I'm not sure how you would know, when making pandoc-fignos==1.0 that it would stop working at pandoc-xnos==0.15, which presumably hadn't been released yet.

**Comment by tomduck** [comment]
Yes, that is exactly the problem.  I will think on it and come up with a solution.

File: content/05.treat.md [hunk 1]
```diff
@@ -165,7 +168,7 @@ When protein structure is available, virtual screening has traditionally relied
 Recently, deep learning approaches have been developed to model protein structure, which is expected to improve upon the simpler drug-target interaction algorithms described above that represent proteins with feature vectors derived from amino acid sequences [@doi:10.1109/BIBM.2014.6999129; @doi:10.1021/acs.jproteome.6b00618].
 
 Structure-based deep learning methods differ in whether they use experimentally-derived or predicted ligand-target complexes and how they represent the 3D structure.
-The Atomic CNN [@arxiv:1703.10603] takes 3D crystal structures from PDBBind [@doi:10.1021/jm048957q] as input, ensuring it uses a reliable ligand-target complex.
+The Atomic CNN [@arxiv:1703.10603] and TopologyNet [@doi:10.1371/journal.pcbi.1005690] take 3D structures from PDBBind [@doi:10.1021/jm048957q] as input, ensuring the ligand-target complexes are reliable.
```
Comment (cgreene) [inline content/05.treat.md]: I think you need to either remove `The` or add `networks` or `models`.

**Comment by agitter** [comment]
> do you know whether the successful builds used the older version of pandoc-fignos

@dhimmel I didn't explicitly change anything in our environment between when the build passed and failed.  However, our [last successful build](https://travis-ci.org/greenelab/deep-review/builds/328442851#L1725) on master did use `pandoc-xnos` version 0.14.

It looks like we have a root cause now and can merge this once we pull in greenelab/manubot-rootstock#99  Thanks for the help @tomduck.

**Comment by dhimmel** [comment]
@agitter, I just merged https://github.com/greenelab/deep-review/pull/778 so you should be good to go.

**Pull request merged**